python/cog/command/ast_openapi_schema.py

```diff
@@ -334,17 +334,24 @@ def to_serializable(val: "AstVal") -> "JSONObject":
 
 def get_value(node: ast.AST) -> "AstVal":
     """Return the value of constant or list of constants"""
-    if isinstance(node, ast.Constant):
-        return node.value
-    # for python3.7, were deprecated for Constant
-    if isinstance(node, (ast.Str, ast.Bytes)):
-        return node.s
-    if isinstance(node, ast.Num):
-        return node.n
+    # Deprecated since version 3.8:
```

Comment:
In this case, do we need to check for constants regardless of python version and then drop down into the version check and check the str/bytes/num?

Reply:
Sorry, this commit shouldn't have been applied